views/scss/qti/_text-entry.scss

```diff
@@ -51,6 +51,7 @@
         line-height: 2.5;
         border: 2px solid black;
         padding: 5px;
-        margin: 0 5px;
+        margin-block: 0;
+        margin-inline: 5px;
```

Comment:
In review mode (can switch renderer [here](https://github.com/oat-sa/tao-item-runner-qti-fe/blob/master/src/runner/rendererStrategies.js#L118-L119)): 

![Screenshot_77](https://github.com/user-attachments/assets/a82485a5-dba6-4b13-8358-a97844f66fdd)
![Screenshot_74](https://github.com/user-attachments/assets/41aab72a-7424-4b37-8be4-a9433c8a4e0e)